### PR TITLE
Remove Interactive EU from frontpage

### PR DESCRIPTION
**Pull request by phillipj**
Since the conference has ended, it's about time to remove the Interactive EU banner from the frontpage?
### BEFORE

screen shot 2016-09-22 at 15 37 16
### AFTER

screen shot 2016-09-22 at 15 38 00

### CODE REVIEW

**Comment by lpinca** [comment]
I think the relevant CSS media queries should also be tweaked.

**Comment by phillipj** [comment]
Thanks for the reminder @lpinca! Did a small CSS refactoring and simplified HTML a bit. Still looks the same as the **after** screenshot, just w/o the redundant code.

File: layouts/index.hbs [hunk 1]
```diff
@@ -83,41 +83,22 @@
...
-      <a class="interactive-item" href="http://events.linuxfoundation.org/events/node-interactive-europe">
-        <img src="/static/images/interactive/nodejs-interactive-hero-banner-left-eu.png"
-          alt="Node.js Interactive Europe, Amsterdam, Netherlands, September 15-18, 2016"/>
-      </a>
-      <span id="logo-center" class="interactive-item">
-        <img src="/static/images/interactive/nodejs-interactive-logo-center.png"
-          alt="Node.js Interactive" />
-      </span>
-      <a class="interactive-item" href="http://events.linuxfoundation.org/events/node-interactive">
+      <img src="/static/images/interactive/nodejs-interactive-logo-center.png" />
```
Comment (lpinca) [inline layouts/index.hbs]: The wrapper span is still required or the image will not be scaled well. This is how it looks in Chrome, responsive mode (Tablet - 768px):

capture

**Comment by lpinca** [comment]
Do we care about IE11? If so can anyone test this on IE11? I remember I spent some time to make this banner work there.

**Comment by phillipj** [comment]
> If so can anyone test this on IE11? I remember I spent some time to make this banner work there.

Scaling of logo and responsiveness looks to be working as expected on my win7/IE11 VM.

**Review by lpinca — APPROVED** [review]
LGTM

**Review by williamkapke — APPROVED** [review]

**Comment by williamkapke** [comment]
Now that this is merged, I realized that the background is Amsterdam and it should probably be Austin. heh

I'll reach out to them and see if they can provide a new image for Austin.

**Comment by lpinca** [comment]
@williamkapke we can cut the background. The right part should be correct, it is huge.

background

**Pull request merged**